app/src/main/java/com/android/streetworkapp/ui/event/EventOverview.kt

```diff
@@ -271,20 +271,20 @@ fun DashBoardBar() {
     NavigationBarItem(
         modifier = Modifier.testTag("detailsTab"),
         icon = { Text("Details") },
-        selected = state == OverviewUiState.Details,
-        onClick = { uiState.value = OverviewUiState.Details })
+        selected = state == DashboardState.Details,
+        onClick = { uiState.value = DashboardState.Details })
 
     NavigationBarItem(
         modifier = Modifier.testTag("participantsTab"),
         icon = { Text("Participants") },
-        selected = state == OverviewUiState.Participants,
-        onClick = { uiState.value = OverviewUiState.Participants })
+        selected = state == DashboardState.Participants,
+        onClick = { uiState.value = DashboardState.Participants })
   }
 }
 
 /** Represents the different states of the event dashboard */
-sealed class OverviewUiState {
-  data object Details : OverviewUiState()
+sealed class DashboardState {
+  data object Details : DashboardState()
 
-  data object Participants : OverviewUiState()
+  data object Participants : DashboardState()
```

Comment:
Question: Why did this sealed class changed?

Reply:
I changed the name of the class to not confuse the overview ui states with the dasboard ui state, i think the naming makes it more clear